invoice-spec.json

```diff
@@ -13,6 +13,22 @@
         "Ärkehålans Aktie-Bolag"
       ]
     },
+    "restricted-string": {
+      "type": "string",
+      "minLength": 1,
+      "maxLength": 50,
+      "pattern": "^(?=.{3,50}$)[a-z0-9](?:-?[a-z0-9]+)*$",
```

Comment:
Won't this restrict the `name` field in headers and row to be lower case? I'm not sure we're complying with that restriction right now.

Reply:
`name-value` objects uses `normal-string` which follows the previous string conventions, and headers is an array of `name-value` objects. Good catch though!